src/geojson-layer.ts

```diff
@@ -1,25 +1,39 @@
 import * as React from 'react';
 import * as MapboxGL from 'mapbox-gl/dist/mapbox-gl';
+const isEqual = require('deep-equal'); //tslint:disable-line
```

Comment:
no typings for deep-equal indeed, we could write some or use `equal.lodash` instead maybe